setup.py

```diff
@@ -52,8 +53,18 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-PACKAGE_DATA = {'pydicom': ['tests/test_files/*', 'tests/charset_files/*']}
+PACKAGE_DATA = {'pydicom': ['tests/test_files/*',
+                            'tests/charset_files/*']}
 REQUIRES = []
+DATA_FILES = []
+
+data_roots = ['pydicom/tests']
+
```

Comment:
Here I made data_roots a list to iterate through with some anticipation that we may want to add other data/ locations in the future.